### PR TITLE
add the Code of Conduct

### PR DESCRIPTION
**Pull request by mr-c**
- [x] verify contact person for reporting CoC violations for the leadership team & contact info. C. Titus Brown has already opted in for me and John Chilton has confirmed that Anton is the correct person for him.

### CODE REVIEW

**Comment by jmchilton** [comment]
This is fantastic, thanks a bunch @mc-c. :+1:.

**Comment by tetron** [comment]
@mr-c this is awesome, some suggestions for clarifications:

```Harassment includes:```
Maybe spell out "includes but is not limited to..."

```
This code of conduct applies to all CWL Project spaces, including the Google
Group, the Gitter chat room, the Google Hangouts chats, both online and
off.
```
Although it already says "both online and off" maybe spell out a little more clearly that this applies equally to in person meetups, workshops, conferences, etc.

```
- Deliberate misgendering or use of ‘dead’ or rejected names.
```
Can we avoid the use of the word 'dead' and just say "former".  The first time I read this I thought it was talking about people who were actually literally dead.

```
 - Physical contact and simulated physical contact (eg, textual descriptions like
+“\*hug\*” or “\*backrub\*”) without consent or after a request to stop.
```
Should be "Physical contact _or_ simulated physical contact"

```
 - Communicating in a ‘tone’ you don’t find congenial
```
"tone" is very vague.  I think you mean something like "Communication you dislike or disagree with that is not judged to be harassment under the above guidelines"

```
 - Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or
assumptions
```
If you pair this with the top line I don't think it means what you intended it to mean:

"The CWL Leadeship Team will not act on complaints criticizing racist, sexist, cissexist, or otherwise oppressive behavior or assumptions."

I think you mean "complaints about criticism of ..." but the sentence is still a little hard to follow.

 ```
To report an issue with anyone on the team you can escalate to...
```

The appropriate contact here would be Ward Vandewege (ward@curoverse.com) instead of Alexander Zaranek.

**Comment by mr-c** [comment]
On Mon, Nov 2, 2015 at 4:01 PM Peter Amstutz <notifications@github.com>
wrote:

> @mr-c <https://github.com/mr-c> this is awesome, some suggestions for
> clarifications:
>
Thank you; and thanks for the detailed review.


> Harassment includes:
> Maybe spell out "includes but is not limited to..."
>
Fixed

> This code of conduct applies to all CWL Project spaces, including the Google
> Group, the Gitter chat room, the Google Hangouts chats, both online and
> off.
>
> Although it already says "both online and off" maybe spell out a little
> more clearly that this applies equally to in person meetups, workshops,
> conferences, etc.
>
Fixed


> - Deliberate misgendering or use of ‘dead’ or rejected names.
>
> Can we avoid the use of the word 'dead' and just say "former". The first
> time I read this I thought it was talking about people who were actually
> literally dead.
>

Ah. It is referring to "deadnaming":
https://www.quora.com/What-is-deadnaming/answer/Nancy-C-Walker
I've added a clarification


>  - Physical contact and simulated physical contact (eg, textual descriptions like
> +“\*hug\*” or “\*backrub\*”) without consent or after a request to stop.
>
> Should be "Physical contact *or* simulated physical contact"
>
Since the heading is "Harassment includes" the "and" is correct.

 - Communicating in a ‘tone’ you don’t find congenial
>
> "tone" is very vague. I think you mean something like "Communication you
> dislike or disagree with that is not judged to be harassment under the
> above guidelines"
>
This is a reference to http://geekfeminism.wikia.com/wiki/Tone_argument
Link added to the text.

>  - Criticizing racist, sexist, cissexist, or otherwise oppressive behavior or
> assumptions
>
> If you pair this with the top line I don't think it means what you
> intended it to mean:
>
> "The CWL Leadeship Team will not act on complaints criticizing racist,
> sexist, cissexist, or otherwise oppressive behavior or assumptions."
>
> I think you mean "complaints about criticism of ..." but the sentence is
> still a little hard to follow.
>
You are right, it doesn't make sense; I'm dropping it.



> To report an issue with anyone on the team you can escalate to...
>
> The appropriate contact here would be Ward Vandewege (ward@curoverse.com)
> instead of Alexander Zaranek.
>
Fixed

**Comment by tetron** [comment]
+1

**Comment by mr-c** [comment]
Thanks!
